src/subscription/checkout/StripeOptions.js

```diff
@@ -3,9 +3,18 @@
  * @param {string} clientSecretId use to initial stripe
  * @returns stripe options
  */
-export const getStripeOptions = (clientSecretId) => ({
+export const getStripeOptions = () => ({
   // Stripe element config
-  clientSecret: clientSecretId,
+  mode: 'subscription',
+  /**
+   * TODO: this should be fetched dynamically
+   * NOTE: this is important to set the right subscription price as defined within stripe subscription
+   * otherwise card details will not load properly
+   * https://dashboard.stripe.com/test/subscriptions/sub_1Mtd3sH4caH7G0X1dvzlMT3o
+   * */
+  amount: 55.00,
+  currency: 'usd', // TODO: this should be fetched dynamically
```

Comment:
This is available in `GET /stripe-checkout` that Chris merged recently, but I noticed it's `'USD'` which it gets from Discovery, I wonder if capitalized will be an issue

Reply:
added `toLowerCase` transformation here -> in my next PR.